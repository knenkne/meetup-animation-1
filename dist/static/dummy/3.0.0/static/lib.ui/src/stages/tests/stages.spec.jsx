import React from 'react'
import renderer from 'react-test-renderer'

import { Stages } from '../stages'

describe('<Stages />', () => {
    it('Рендерит children', () => {
        const tree = renderer
            .create(
                <Stages translations={{ tooltip: 'Вы здесь' }}>
                    <li />
                </Stages>
            )
            .toJSON()

        expect(tree).toMatchSnapshot()
    })
})
