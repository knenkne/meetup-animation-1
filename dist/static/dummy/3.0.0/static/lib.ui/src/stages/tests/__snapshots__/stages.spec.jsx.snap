// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Stages /> Рендерит children 1`] = `
<ul
  className="stages centered"
>
  <li
    translations={
      Object {
        "tooltip": "Вы здесь",
      }
    }
    value={1}
  />
</ul>
`;
